### ISSUE TITLE
Make a nice terminal interface / clean up Ice's terminal output

### ISSUE THREAD

**Issue by scottrice** [scottrice/Ice#368]
Rather than just print out a subset of my log messages, I think it would be better to have an actual terminal interface using python's curses library.

Ideas for things that I could put in this new interface:

* The location of the Steam install we're using (whatever directory contains `userdata`)
* The emulators (and maybe a check or x based on whether a file exists at that location?)
* The consoles and what emulators they're using
* How many ROMs were found for each console
* The games which have images and those that don't (move them from one list to another as we find images).

**Comment by ProfessorKaos64** [comment]
Good idea, I like it. Maybe time stamps of import, successful/unsuccessful imports, which ones were duplicate image downloads etc.
